src/Locales/pl/Pode.psd1

```diff
@@ -0,0 +1,287 @@
+    daysInMonthExceededExceptionMessage                               = '{0} ma tylko {1} dni, ale podano {2}.'
+    nonEmptyScriptBlockRequiredForCustomLoggingExceptionMessage       = 'Metoda niestandardowego rejestrowania wymaga niepustego ScriptBlock.'
+    encodingAttributeOnlyAppliesToMultipartExceptionMessage           = 'Atrybut kodowania dotyczy tylko ciał żądania typu multipart i application/x-www-form-urlencoded.'
+    suppliedDateBeforeScheduleStartTimeExceptionMessage               = 'Podana data jest wcześniejsza niż czas rozpoczęcia harmonogramu o {0}'
+    unlockSecretRequiredExceptionMessage                              = "Właściwość 'UnlockSecret' jest wymagana przy używaniu Microsoft.PowerShell.SecretStore"
+    noLogicPassedForMethodRouteExceptionMessage                       = '[{0}] {1}: Brak logiki przekazanej.'
+    bodyParserAlreadyDefinedForContentTypeExceptionMessage            = 'Parser treści dla typu zawartości {0} jest już zdefiniowany.'
+    invalidJwtSuppliedExceptionMessage                                = 'Dostarczono nieprawidłowy JWT.'
+    sessionsRequiredForFlashMessagesExceptionMessage                  = 'Sesje są wymagane do używania wiadomości Flash.'
+    semaphoreAlreadyExistsExceptionMessage                            = "Semafor o nazwie '{0}' już istnieje."
+    invalidJwtHeaderAlgorithmSuppliedExceptionMessage                 = 'Dostarczono nieprawidłowy algorytm nagłówka JWT.'
+    oauth2ProviderDoesNotSupportPasswordGrantTypeExceptionMessage     = "Dostawca OAuth2 nie obsługuje typu 'password' wymaganego przez InnerScheme."
+    invalidAliasFoundExceptionMessage                                 = 'Znaleziono nieprawidłowy alias {0}: {1}'
+    scheduleDoesNotExistExceptionMessage                              = "Harmonogram '{0}' nie istnieje."
+    accessMethodNotExistExceptionMessage                              = 'Metoda dostępu nie istnieje: {0}'
+    oauth2ProviderDoesNotSupportCodeResponseTypeExceptionMessage      = "Dostawca OAuth2 nie obsługuje typu odpowiedzi 'code'."
+    untestedPowerShellVersionWarningMessage                           = '[OSTRZEŻENIE] Pode {0} nie był testowany na PowerShell {1}, ponieważ nie był dostępny, gdy Pode został wydany.'
+    secretVaultAlreadyRegisteredAutoImportExceptionMessage            = "Skarbiec z nazwą '{0}' został już zarejestrowany podczas automatycznego importowania skarbców."
+    schemeRequiresValidScriptBlockExceptionMessage                    = "Dostarczony schemat dla walidatora uwierzytelniania '{0}' wymaga ważnego ScriptBlock."
+    serverLoopingMessage                                              = 'Pętla serwera co {0} sekund'
+    certificateThumbprintsNameSupportedOnWindowsExceptionMessage      = 'Odciski palców/nazwa certyfikatu są obsługiwane tylko w systemie Windows.'
+    sseConnectionNameRequiredExceptionMessage                         = "Wymagana jest nazwa połączenia SSE, z -Name lub $`$WebEvent.Sse.Name"
```

Comment:
Typo in the escaping of "$" for "$WebEvent", there's an erroneous "$" in front of the backtick